README.md

```diff
@@ -94,17 +94,20 @@ package / module name it follows python style conventions of being short and all
 ## Testing
 
 All tests are written in [pytest](https://docs.pytest.org/en/7.2.x/index.html).
-End-to-end tests are written with the [playwright plugin of pytest](https://playwright.dev/python/docs/intro).
-To provide a test-database for some tests we use [pytest-django](https://pytest-django.readthedocs.io/en/latest/index.html).
+End-to-end tests are written with the [playwright plugin of pytest](https://playwright.dev/python/docs/intro). <br>
+The end-to-end tests use [pytest-django](https://pytest-django.readthedocs.io/en/latest/index.html) to set up a live server and
+provide a test-database that is cleaned after each test. <br>
+In order to automatically test the docker setup, there is also a smoke test in `e2e_tests/test_deployed` that requires starting the application with the local
+configuration and creating the database with migrations applied and `e2e_tests/database/test_database.json` loaded.
 
+#### Install playwright
 The first time playwright is used, let it download the tools it needs with:
 
 ```shell
 playwright install
 ```
 
-*Start the docker container with the local config*, then execute tests with
-
+#### Run the tests
 ```shell
 # run all tests
```

Comment:
would be nice to add `--ignore e2e_tests/test_deployed.py` here. Better yet, rename the file to e.g. `xtest_deployed.py` and state somewhere, how to start it explicitly, and add it explicitly to the CI.

Reply:
Added #153

Reply:
I will add the `--ignore e2e_tests/test_deployed.py` because I do not want to break with pytest's naming conventions. For grouping tests pytest also has marks https://docs.pytest.org/en/7.1.x/how-to/mark.html. But it seems like overkill at the moment as we can easily exclude the deployed tests with the option you provided.